internal/engine/compiler/compiler_controlflow_test.go

```diff
@@ -310,120 +310,109 @@ func TestCompiler_compileBrTable(t *testing.T) {
 		require.Equal(t, expValue, env.stackTopAsUint32())
 	}
 
-	getBranchTargetDropFromFrameID := func(frameid uint32) *wazeroir.BranchTargetDrop {
-		return &wazeroir.BranchTargetDrop{
-			Target: wazeroir.Label{FrameID: frameid, Kind: wazeroir.LabelKindHeader},
-		}
+	getBranchLabelFromFrameID := func(frameid uint32) uint64 {
+		return uint64(wazeroir.NewLabelID(wazeroir.LabelKindHeader, frameid))
+	}
+
+	newOperationBrTableWithDefaultRanges := func(us ...uint64) wazeroir.UnionOperation {
+		targetRanges := make([]*wazeroir.InclusiveRange, len(us))
```

Comment:
wonder if we still need this?

Reply:
ps: I wouldn't worry too much about this because it's a test; point is, either we allow to pass a nil slice here or we'll have to fix where this is handled (probably at the interpreter site and the compilation site)

Reply:
heh I think I managed to fix it :)